service/src/test/java/org/sunbird/learner/actors/skill/UserSkillManagementActorTest.java

```diff
@@ -48,7 +48,7 @@
   private static final String ENDORSED_USER_ID = "someEndorsedUserId";
   private static final String ENDORSE_SKILL_NAME = "C";
   private static List<String> skillsList = new ArrayList<>();
-  private static FiniteDuration duration = duration("10 second");
+  private static FiniteDuration duration = duration("10000 second");
```

Comment:
What time it generally takes -> we cannot wait for 10000 second, that is too large interval - try out smaller intervals, and see if that works?